Competition2017/src/competition/subsystems/shooter_wheel/ShooterWheelSubsystem.java

```diff
@@ -33,9 +38,22 @@ public ShooterWheelSubsystem(
                 propManager);
         log.info("Creating");
         this.side = side;
+        flushToBoilerTargetSpeed = 
+                propManager.createPersistentProperty(side + " flush to boiler target speed", 3500);
     }
+    
     public RobotSide getSide(){
         return side;
     }
+    
+    public void setTargetSpeedForRange(ShooterRange range) {
+        switch (range) {
+        case FlushToBoiler:
```

Comment:
Why use an `enum` as the only means of specifying the distance? This is pretty inflexible. I'd prefer having a method that translated the enum to distance, and then a separate method you fed that to. Maybe with an overload for the current behavior.